code/game/objects/items/weapons/clown_items.dm

```diff
@@ -238,43 +240,80 @@
 
 	if(isitem(target))
 		var/obj/item/target_item = target
+		if(target_item.current_glue_state != GLUE_STATE_NONE) //Check to see if its glued first.
+			to_chat(user,"<span class='warning'>It already has glue on it!</span>")
+			return
 		if(target_item.abstract) //Can't glue TK grabs, grabs, offhands!
 			return
 
-	to_chat(user,"<span class='info'>You gently apply the whole [src] to \the [target].</span>")
-	spent = 1
+	to_chat(user,"<span class='info'>You put some glue on \the [target].</span>")
+	if(uses < 0) //For superglue
```

Comment:
this is weird, just make a proc/use() and call it here, you could even take a num arg to decide how much glue to spend, just in case you wanted to add the ability to glue something bigger for more glue in the future. For superglue, have it set to spent and for normal glue, decrement uses.

I'm still not entirely sure why you have `spent` and `uses` - can you explain your reasoning? Why not just give superglue `uses = 1`?

Reply:
I have no excuse for this one, don't know how I didn't think of that.